Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/other-functions/obj-description.adoc

= obj_description()
:description: The link:https://www.postgresql.org/docs/current/functions-info.html#FUNCTIONS-INFO-COMMENT[obj_description()^] is a comment information function that 
:page-topic-type: reference

The link:https://www.postgresql.org/docs/current/functions-info.html#FUNCTIONS-INFO-COMMENT[obj_description()^] is a comment information function that returns the comment associated with a specific database object.

== Syntax

The syntax for the `obj_description()` function is as follows:

....
obj_description (object_oid, catalog_name) → NULL
....

== Parameters

> The following parameters are required to execute this function:

Reviewer: ```suggestion
These parameters are required to execute this function:
```
